src/main/java/pl/confitura/jelatyna/participant/Controller.java

```diff
@@ -0,0 +1,37 @@
+package pl.confitura.jelatyna.participant;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+import static org.springframework.web.bind.annotation.RequestMethod.PUT;
+
+@RestController("participantController")
+@RequestMapping("/api/participant")
+public class Controller {
```

Comment:
Słusznie.. chciałem to zmienić. Nie możemy po prostu wystawiać repo. do okoła zapisywania,pobierania itd. będą się działy różne inne wydarzenia. np. wysyłka maili. więc powinniśmy mieć `Service`